### ISSUE TITLE
Make Sentry Recipe (convert existing example code into a recipe)

### ISSUE THREAD

**Issue by flybayer** [blitz-js/blitz#1188]
### What do you want and why?

Make [a blitz recipe](https://blitzjs.com/docs/writing-recipes) for Sentry!

This repo shows how to integrate Sentry: https://github.com/flybayer/blitz-sentry

So need to convert that into a recipe.

**Comment by enricoschaaf** [comment]
Would it be wise to swap out sentry/node with sentry/react?

**Comment by flybayer** [comment]
@enricoschaaf maybe, idk! Not sure what the difference is

**Comment by enricoschaaf** [comment]
Don't know much but it's seems to be a wrapper around sentry/browser. What's seemed interesting to me is that it supports error boundaries. https://docs.sentry.io/platforms/javascript/guides/react/

**Comment by flybayer** [comment]
@enricoschaaf ok, probably `react-error-boundary` is better than what they have unless there's is built on this 🤔

**Comment by enricoschaaf** [comment]
I think they would serve diffrent purposes you would use react-error-boundary for errors that you dont want to get sent to sentry because the sentry one relays them. But I dont know it thats any use🤷‍♂️

**Comment by flybayer** [comment]
Ok yeah. Sounds more complex to have an additional error boundary than what's in the example 🤷

**Comment by engelkes-finstreet** [comment]
@flybayer I have some problems to correctly replace the content of `blitz.config.js` with codeshift. Would it be viable to replace the whole file?

**Comment by flybayer** [comment]
@engelkes-finstreet no, because that would overwrite any stuff the user has customized already. I'm sure it's possible to do this with codeshift, just need to figure out the way 😅

**Comment by engelkes-finstreet** [comment]
Hm I cannot figure it out. Is there someone who is a codeshift expert and could help me out with this?
